File: MyApplication3/MyApplication3/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;
import androidx.constraintlayout.widget.ConstraintLayout;

import android.graphics.Color;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.LinearLayout;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView atv01=(TextView) findViewById(R.id.tv01);
        Button abtn01=(Button) findViewById(R.id.btn01);
        Button abtn02=(Button) findViewById(R.id.btn02);
        ConstraintLayout alayout=(ConstraintLayout) findViewById(R.id.layout);
        //增加监听器及方法
        Button.OnClickListener Listener =new Button.OnClickListener(){
//            @SuppressWarnings("deprecation")
            public void onClick(View view) {
                // TODO Auto-generated method stub
                switch(view.getId()){
                    case R.id.btn01:
                        alayout.setBackgroundColor(Color.parseColor("#FFFF33"));
                        break;
                    case R.id.btn02:
                        //调用布局，更改布局颜色。使用系统中的color工具类parseColor设置
                        alayout.setBackgroundColor(Color.parseColor("#FF0000"));
                        break;
                }
            }};
        //给按钮绑定监听器，必须绑定监听器，否则没有效果
        abtn01.setOnClickListener(Listener);
        abtn02.setOnClickListener(Listener);
    }

}

File: CODE/MyApplicationMain/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import android.annotation.SuppressLint
import android.content.Intent
import android.os.Bundle
import android.os.Looper
import android.view.MenuItem
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.airbnb.lottie.LottieAnimationView
import com.google.android.material.bottomnavigation.BottomNavigationView
import java.util.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val actionBar = supportActionBar
        actionBar?.hide()

        val textView1 = findViewById<TextView>(R.id.introductionView1)
        val textView2 = findViewById<TextView>(R.id.introductionView2)
        val lottieAnimationView = findViewById<LottieAnimationView>(R.id.cartoon)

        lottieAnimationView.animate().translationY(-1000f).setDuration(1000).setStartDelay(2000)
        textView1.animate().translationY(1000f).setDuration(1000).setStartDelay(2000)
        textView2.animate().translationY(1000f).setDuration(1000).setStartDelay(2000)

        val timer = Timer()
        val timerTask: TimerTask = object : TimerTask() {
            override fun run() {
                val bottomNavigationView: BottomNavigationView
                bottomNavigationView = findViewById(R.id.bottomNavigationView)

                runOnUiThread {
                    bottomNavigationView.visibility = View.VISIBLE
                }

                if (savedInstanceState == null) {
                    supportFragmentManager.beginTransaction().replace(R.id.fragment, Home()).commit()
                }

                bottomNavigationView.setOnNavigationItemSelectedListener(BottomNavigationView.OnNavigationItemSelectedListener { item: MenuItem ->
                    val fragment: Fragment = when (item.itemId) {
                        R.id.home_ -> Home()
                        R.id.communication_ -> MainCommunication()
                        R.id.me_ -> MainUser()
                        else -> return@OnNavigationItemSelectedListener false
                    }
                    if (fragment != null) {
                        supportFragmentManager.beginTransaction().replace(R.id.fragment, fragment).commit()
                        return@OnNavigationItemSelectedListener true
                    }
                    false
                })
            }
        }
        timer.schedule(timerTask, 3400)
    }
}
